### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by Creeplays** [assusdan/drovos#3]
I am running drovos in production for 3 months, but i see one bug: No interrupts are handled.
Will that bug get fixed?
May be move drovos to Drov. Inc?

**Comment by assusdan** [comment]
Yes it'll be fixed in 2-3 months. As soon as I'll finish my current project.

**Comment by Creeplays** [comment]
@assusdan, thats awesome! I really like performance of this OS!
